_papers/thesis-report/additional/extended-abstract.typ

```diff
@@ -100,11 +100,15 @@ The effectiveness of reading data in a decentralized environment has been increa
 In this work, we will similarly research how we can abstract data updates by using a query abstraction layer.
 The current (draft) Solid specification~@bib1:solid-spec describes each data store, or pod, as a document oriented interface where a user decides for each document who can access that document.
 Our goal is thus to create a query engine that effectively decides what document a resource should be stored in. Easily eliminating the access-path data dependency.
-We hypothesize that such a query engine has a 2x overhead in the number of HTTP requests and a 4x overhead in the execution time compared to a query engine that requires the user to configure the document explicitly. Such an overhead is acceptable since write speeds are, in contrast with read speeds, often not critical.  
+We hypothesize that such a query engine has a 2x overhead in the number of HTTP requests and a 4x overhead in the execution time compared to a query engine that requires the user to configure the document explicitly. Such an overhead is acceptable since write speeds are, in contrast with read speeds, often not critical.
+
+// TODO: Can you in a few words explain why this overhead is acceptable?"
```

Comment:
I'm unsure. They might not be for some, they might be for others? I could say that in the context of distributed databases a sync often happens in the background?

Reply:
> I could say that in the context of distributed databases a sync often happens in the background?

Yes, I share that reasoning.